src/components/Form/Debloat.md

```diff
@@ -0,0 +1,82 @@
+```jsx
+import formFields from './DebloatStub.js'
+
+import {
+  TitleLarge,
+  TextInput,
+  Spacer,
+  Button,
+  InfoMessage,
+  ZipInput,
+  EmailInput,
+  Form,
+  PasswordInput,
+} from '../index'
+
+let count = 1;
+
+;<Form
```

Comment:
the leading ; is just an annoyance we have to do for react-styleguidist so it doesn't get confused